lib/ui/src/modules/ui/components/theme.js

```diff
@@ -1,7 +1,7 @@
 export const baseFonts = {
   fontFamily: `
-    -apple-system, ".SFNSText-Regular", "San Francisco", "Roboto",
-    "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif
+    -apple-system, "BlinkMacSystemFont", ".SFNSText-Regular", "San Francisco", "Roboto",
```

Comment:
Well we've discussed in slack in the context of menu items jumping when they go from bold to normal. I can extract it to a separate PR if needed